gsuite/provider.go

```diff
@@ -38,6 +38,11 @@ func Provider() *schema.Provider {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"timeout_minutes": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  1, // 1 + (n*2) roof 16 = 1+2+4+8+16 = 31 seconds, 1 min should be "normal" operations
```

Comment:
should it default to 4 to stay consistent with the previous hard-coded default?

Reply:
I set this to 4 minutes, more or less baselessly and randomly, to fight some timeouts for a specific project, the 1 minute (as suggested by the comment) is based on _maths_